src/utils/es-wrapper.js

```diff
@@ -1,12 +1,21 @@
-const elasticsearch = require('elasticsearch')
-const httpAwsEs = require('http-aws-es')
+const { Client } = require('@elastic/elasticsearch')
+const { createAWSConnection, awsGetCredentials } = require('@acuris/aws-es-connection')
```

Comment:
Instead of using this library I would recommend to move the logic behind it here. I suppose there will be few lines of code.

Reply:
The original `es-wrapper` uses a simple external lib `http-aws-es`, too. So the number of external deps does not increase. I would leave it because bugs/improvements can be fixed/implemented in the external lib more likely than in few lines here. But last word to you @matrus2!